projects/plutus/src/x86/ltc_adc.c

```diff
@@ -0,0 +1,62 @@
+#include "ltc_adc.h"
+
+// x86 module for mocking current sense data
+
+#include "critical_section.h"
+#include "current_sense.h"
+#include "ltc2484.h"
+
+static int32_t s_test_voltage = 0;
+
+static void prv_ltc_adc_read(SoftTimerID timer_id, void *context) {
+  LtcAdcStorage *storage = (LtcAdcStorage *)context;
+  CurrentSenseStorage *current_sense = (CurrentSenseStorage *)context;
```

Comment:
How can you have two different structs from the same context?

Reply:
Ran out of lines